language/src/name_resolution.rs

```diff
@@ -551,7 +551,7 @@ fn process_decl_item(
                                     },
                                     None,
 				),
-				tag : ItemTag::Code
+				tags : ItemTagSet([ItemTag::Code].iter().cloned().collect())
```

Comment:
Wouldn't `HashSet::new(ItemTag::Code)` be nicer ?

Reply:
Oh, I am a bit new to rust, so it would probably be a lot nicer :) , I will clean that up immediately.

Reply:
Hmm, can you do `HashSet::new` with arguments, cargo complains about there being an argument.

Reply:
Oh yeah sorry it's `HashSet::singleton(ItemTag::Code)`. You can lookup the API of `HashSet` here: https://docs.rs/im/12.2.0/im/struct.HashSet.html

Reply:
I get an error with `function or associated item not found in `im::HashSet<_, _>`, however I can just use `HashSet::new().update()`.

Reply:
It has been ranamed to `unit` in the newest version of rust [https://docs.rs/im/15.0.0/im/struct.HashSet.html](url).